debug/atmosphere.html

```diff
@@ -222,6 +253,19 @@
         "space-color": 'rgba(' + guiParams.spaceColor[0] + ', ' + guiParams.spaceColor[1] + ', ' + guiParams.spaceColor[2] + ', ' + guiParams.spaceAlpha + ')',
         "star-intensity": guiParams.starIntensity
     });
+
+    map.addSource('mapbox-dem', {
+        'type': 'raster-dem',
+        'url': 'mapbox://mapbox.terrain-rgb',
+        'tileSize': 512,
+        'maxzoom': 14
+    });
+
+    map.setTerrain({
+        'source': 'mapbox-dem',
+        'exaggeration': 1.5
+    });
```

Comment:
Do you think it's necessary that we add terrain on this one or would it be ok to keep it light?

Reply:
Ok I just tried the example and the zoom-in shows the terrain as it's at high zoom. Disregard that comment